### DISCUSSION TITLE
End-to-end OCR method support？

### DISCUSSION THREAD

**Discussion by htaoruan** [open-mmlab/mmocr#1738] [General]
Will the end-to-end OCR method be supported later? What is the approximate time frame?

**Comment by gaotongxiao** [comment]
[ABCNet](https://github.com/open-mmlab/mmocr/blob/1.x/projects/ABCNet/README.md) is already supported in `1.x` branch. ABCNet v2 and SPTS will be released by the end of Feb.

**Reply by htaoruan** [reply]
OK thankyou

**Reply by htaoruan** [reply]
Hello, In 1.x branch, no configuration file of ABCnet?

**Reply by Harold-lkk** [reply]
https://github.com/open-mmlab/mmocr/blob/dev-1.x/projects/ABCNet/config/abcnet/abcnet_resnet50_fpn_500e_icdar2015.py

**Reply by htaoruan** [reply]
OK  thank you
